runtime-environment-spec/polkadot_re_spec.tm

```diff
@@ -1778,21 +1778,23 @@
 
   <subsubsection|Inherents>
 
-  Block inherent data represent the totality of inherent extrinsics included
-  in each block. In general, these data are collected or generated by
-  Polkadot RE and handed to Runtime for inclusion in the block. Table
-  <reference|tabl-inherent-data> lists these inherent data, their
-  identifiers, and types.
-
-  <\big-table|<tabular|<tformat|<cwith|1|-1|1|-1|cell-tborder|0ln>|<cwith|1|-1|1|-1|cell-bborder|0ln>|<cwith|1|-1|1|-1|cell-lborder|0ln>|<cwith|1|-1|1|-1|cell-rborder|0ln>|<cwith|5|5|1|-1|cell-bborder|1ln>|<cwith|1|-1|1|1|cell-lborder|0ln>|<cwith|1|-1|3|3|cell-rborder|0ln>|<cwith|1|1|1|-1|cell-tborder|1ln>|<cwith|1|1|1|-1|cell-bborder|1ln>|<cwith|2|2|1|-1|cell-tborder|1ln>|<cwith|1|1|1|1|cell-lborder|0ln>|<cwith|1|1|3|3|cell-rborder|0ln>|<table|<row|<cell|Identifier>|<cell|Type
-  >|<cell|Description>>|<row|<cell|timstap0>|<cell|u64>|<cell|Unix epoch time
-  in number of seconds>>|<row|<cell|babeslot>|<cell|u64>|<cell|Babe Slot
-  Number<rsup|<reference|defn-epoch-slot>>>>|<row|<cell|finalnum>|<cell|compact
-  integer<math|<rsup|<reference|defn-sc-len-encoding>>>>|<cell|Finality
-  tracker of the header number<math|<rsup|<reference|defn-block-header>>>>>|<row|<cell|uncles00>|<cell|array
-  of block headers<math|<rsup|>>>|<cell|Provides a list of potential uncle
-  block headers<math|<rsup|<reference|defn-block-header>>> for a given
-  block>>>>>>
+  Block inherent data represents the totality of inherent extrinsics included
```

Comment:
We need to keep a convention sheet keeping track of conventions like "data is singular".